docker-cron/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:latest
```

Comment:
`FROM golang:alpine`
can be used directly to minimize the number of layers in the image.